keyboards/cannonkeys/tomak_type_a/info.json

```
@@ -0,0 +1,256 @@
+            {"matrix": [9, 0], "x": 95, "y": 40, "flags": 4},
+            {"matrix": [9, 1], "x": 107, "y": 40, "flags": 4},
+            {"matrix": [9, 2], "x": 120, "y": 40, "flags": 4},
+            {"matrix": [9, 3], "x": 132, "y": 40, "flags": 4},
+            {"matrix": [9, 4], "x": 144, "y": 40, "flags": 4},
+            {"matrix": [9, 5], "x": 156, "y": 40, "flags": 4},
+            {"matrix": [9, 6], "x": 176, "y": 40, "flags": 5},
+            {"matrix": [10, 7], "x": 184, "y": 52, "flags": 5},
+            {"matrix": [10, 6], "x": 173, "y": 52, "flags": 5},
+            {"matrix": [10, 6], "x": 167, "y": 52, "flags": 5},
+            {"matrix": [10, 5], "x": 150, "y": 52, "flags": 4},
+            {"matrix": [10, 4], "x": 138, "y": 52, "flags": 4},
+            {"matrix": [10, 3], "x": 126, "y": 52, "flags": 4},
+            {"matrix": [10, 2], "x": 114, "y": 52, "flags": 4},
+            {"matrix": [10, 1], "x": 101, "y": 52, "flags": 4},
+            {"matrix": [10, 0], "x": 89, "y": 52, "flags": 4},
+            {"matrix": [11, 0], "x": 95, "y": 64, "flags": 4},
+            {"matrix": [11, 2], "x": 115, "y": 64, "flags": 4},
+            {"matrix": [11, 4], "x": 137, "y": 64, "flags": 5},
+            {"matrix": [11, 5], "x": 152, "y": 64, "flags": 5},
+            {"matrix": [11, 6], "x": 167, "y": 64, "flags": 5},
+            {"matrix": [11, 7], "x": 183, "y": 64, "flags": 5},
+            {"matrix": [11, 8], "x": 199, "y": 64, "flags": 5},
+            {"matrix": [11, 9], "x": 212, "y": 64, "flags": 5},
+            {"matrix": [11, 10], "x": 224, "y": 64, "flags": 5},
+            {"matrix": [10, 9], "x": 212, "y": 52, "flags": 5},
+            {"matrix": [8, 8], "x": 199, "y": 27, "flags": 5},
+            {"matrix": [8, 9], "x": 212, "y": 27, "flags": 5},
+            {"matrix": [8, 10], "x": 224, "y": 27, "flags": 5},
+            {"matrix": [7, 10], "x": 224, "y": 15, "flags": 5},
+            {"matrix": [7, 9], "x": 212, "y": 15, "flags": 5},
+            {"matrix": [7, 8], "x": 199, "y": 15, "flags": 5},
+            {"matrix": [6, 8], "x": 199, "y": 0, "flags": 5},
+            {"matrix": [6, 9], "x": 212, "y": 0, "flags": 5},
+            {"matrix": [6, 10], "x": 224, "y": 0, "flags": 5}
+        ],
+        "split_count": [41, 59]
+    },
+    "split": {
+        "enabled": true,
+        "matrix_pins": {
+            "right": {
+                "cols": ["GP21", "GP22", "GP23", "GP24", "GP25", "GP26", "GP0", "GP17", "GP1", "GP2", "GP3"],
+                "rows": ["GP11", "GP10", "GP9", "GP8", "GP5", "GP4"]
+            }
+        },
+        "soft_serial_pin": "GP6",
+        "transport": {
+            "protocol": "serial"
+        }
+    },
+    "tags": ["split", "RP2040"],
+    "url": "https://cannonkeys.com/",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0027",
+        "vid": "0xCA04"
+    },
+    "ws2812": {
+        "pin": "GP16"
```

migrated from `rules.mk`
```suggestion
        "pin": "GP16",
        "driver": "vendor"
```